src/react/databrowser/objects/details/Properties.jsx

```diff
@@ -61,14 +67,53 @@ function Properties({ objectMetadata }: Props) {
               <Clipboard text={objectMetadata.eTag} />{' '}
             </T.ExtraCell>
           </T.Row>
+          <T.Row>
+            <h3> Data Protection </h3>
+          </T.Row>
           <T.Row>
             <T.Key> Lock </T.Key>
             <T.Value>
-              {objectMetadata.lockStatus === 'LOCKED' && <>Locked ({objectMetadata.objectRetention.mode.toLowerCase()})<br />until {objectMetadata.objectRetention.retainUntilDate}</>}
-              {objectMetadata.lockStatus === 'RELEASED' && `Released - since ${objectMetadata.objectRetention.retainUntilDate}`}
+              {objectMetadata.lockStatus === 'LOCKED' && (
+                <>
+                  Locked ({objectMetadata.objectRetention.mode.toLowerCase()})
+                  <br />
+                  until {objectMetadata.objectRetention.retainUntilDate}
+                </>
+              )}
+              {objectMetadata.lockStatus === 'RELEASED' &&
+                `Released - since ${objectMetadata.objectRetention.retainUntilDate}`}
               {objectMetadata.lockStatus === 'NONE' && 'No retention'}
             </T.Value>
           </T.Row>
+          {/* Display the legal hold when the object lock is enabled at bucket level */}
+          {objectMetadata.lockStatus !== 'NONE' && (
+            <T.Row>
+              <T.Key>Legal Hold</T.Key>
+              <T.Value>
+                <Toggle
+                  toggle={isLegalHoldEnabled}
+                  label={isLegalHoldEnabled ? 'Active' : 'Inactive'}
+                  onChange={() =>
+                    dispatch(
+                      putObjectLegalHold(
+                        objectMetadata.bucketName,
+                        objectMetadata.objectKey,
+                        objectMetadata.versionId,
+                        !isLegalHoldEnabled,
+                        prefixWithSlash,
+                      ),
+                    )
+                  }
+                />
+                {isLegalHoldEnabled && (
+                  <i
+                    className="fas fa-balance-scale"
+                    style={{ paddingLeft: '5px' }}
```

Comment:
shouldn't we use 0.5rem instead ? Maybe zenko is not yet ready for the rem unit conversion, I'm just wonering

Reply:
I actually changed in the later commit with `spacing`
```
const Icon = styled.i`
  margin-left: ${spacing.sp4};
`;
```